### PR TITLE
r.topidx: Update the manual with info about unsupported lat/long locations

### PR DESCRIPTION
**Pull request by HuidaeCho**
This PR addresses #3475.

### CODE REVIEW

File: raster/r.topidx/r.topidx.html [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
+<p>Input maps may have NULL values.  For example, if you have a MASK for a
+watershed (basin map from <em>r.water.outlet</em>), the following command will
+create a masked elevation map (belev):
 <div class="code"><pre>
 r.mapcalc "belev = if(isnull(basin), basin, elev)"
 </pre></div>
 <p>
 <em>r.stats -Anc</em> prints out averaged statistics for topographic index.
 
+<p>Unprojected lat/long locations are not supported.
```
Comment (echoix) [inline raster/r.topidx/r.topidx.html:22]: So what would I, user, need to do then?

**Review by echoix — APPROVED** [review]
Other than that, it's good!

File: raster/r.topidx/r.topidx.html (hunk 1 above)
Comment (HuidaeCho) [reply raster/r.topidx/r.topidx.html:22]: They need to create a new projection location (project now) and import data into that project. I can add this info, but there are more modules that don't support lat/long projections.

Comment (HuidaeCho) [reply raster/r.topidx/r.topidx.html]: Also, I'm not sure how much detail we want to add to individual module manuals. I think this reprojection info should be more global. Anyway, I've added more info.

Comment (echoix) [reply raster/r.topidx/r.topidx.html]: Y'a I know a manual isn't the place for a tutorial... But it's also the only place you go when you want to use a module. Something simple like: (the thing) must be (in a) projected (the good term)...

**Review by echoix — APPROVED** [review]

**Comment by echoix** [comment]
@neteler About the milestone here, since it was merged in main, wouldn't it be 8.4.0? Yes, I see it was backported (without a label), but in that case can a PR have more than one milestone?

**Comment by neteler** [comment]
> About the milestone here, since it was merged in main, wouldn't it be 8.4.0? 

Well, so far we applied the milestone in which the fix is available "first" (so: lowest version number which contains the PR).

> Yes, I see it was backported (without a label), but in that case can a PR have more than one milestone?

To my knowledge only one milestone is possible.

**Pull request merged**